electricitymap/contrib/lib/models/events.py

```diff
@@ -382,10 +400,12 @@ def to_dict(self) -> dict[str, Any]:
 class TotalProduction(Event):
```

Comment:
We should also consider renaming this to TotalGeneration IMO so it's less likely to be confused with the production breakdown.